### PR TITLE
Implemented Character Length Validation on Feedback Message

### PR DESCRIPTION
**Pull request by ananyag309**
solved #218 

 imposed a character length validation that requires feedback messages to have a minimum of at least [insert minimum character length, e.g., 5 or 10] characters. This will ensure that submitted feedback is meaningful and constructive.

image

### CODE REVIEW

File: frontend/common/validation.js [hunk 1]
```diff
@@ -117,4 +117,25 @@ const validate = {
...
+      };
+    } else {
+      return { feedback: false, feedbackError: false };
+    }
+  },
+};
+
+export default validate;
+
 export default validate;
```
Comment (AlfiyaSiddique) [inline frontend/common/validation.js:141]: @ananyag309 this is already exported, please remove one

**Comment by ananyag309** [comment]
@AlfiyaSiddique done

**Comment by AlfiyaSiddique** [comment]
@ananyag309 it is still failing there is a syntax error in the validation.js file.
May be an extra bracket please find it and fix it.

**Comment by ananyag309** [comment]
@AlfiyaSiddique please check i made some changes

**Comment by AlfiyaSiddique** [comment]
@ananyag309 it's deployed now but there is an error in your change which is crashing the app when clicking on Feedback button, the error is as follows:
image

Can you please fix this
you can also check this on deploy preview link

**Comment by AlfiyaSiddique** [comment]
@ananyag309 any updates

**Comment by ananyag309** [comment]
will fix it today

**Pull request closed without merging**